app/assets/stylesheets/base.scss

```diff
@@ -1,13 +1,16 @@
 @import "bootstrap-slider/dist/css/bootstrap-slider";
-@import "bootstrap-tagsinput";
 @import "variables"; // bootstrap variable overrides
-@import "blacklight_heatmaps/default";
+@import "modules/blacklight_heatmaps";
 @import "bootstrap/scss/bootstrap";
-@import "sir-trevor/main";
-@import "openseadragon";
-@import "blacklight";
-@import "blacklight_gallery";
-@import "spotlight";
+@import "modules/openseadragon";
+@import "blacklight-frontend/app/assets/stylesheets/blacklight/blacklight";
+@import "blacklight-gallery/app/assets/stylesheets/blacklight_gallery/gallery";
+@import "blacklight-gallery/app/assets/stylesheets/blacklight_gallery/masonry";
+@import "blacklight-gallery/app/assets/stylesheets/blacklight_gallery/slideshow";
+@import "blacklight-gallery/app/assets/stylesheets/blacklight_gallery/osd_viewer";
+@import "spotlight-frontend/app/assets/stylesheets/spotlight/spotlight";
+@import "spotlight-frontend/vendor/assets/stylesheets/bootstrap-tagsinput";
```

Comment:
This needs to be above `spotlight/spotlight` or the width of the tag input is wrong on the item edit page.